src/stores/tagsStore.js

```diff
@@ -16,15 +17,28 @@ export const useTagsStore = defineStore('tagsStore', {
 			if (!this.tags.includes(tagName)) {
 				this.tags = [ ...this.tags, tagName ];
 				localStorage.setItem(LOCAL_STORAGE_TAGS_KEY, JSON.stringify(this.tags));
+				this.updateUrl();
 			}
 		},
 		unSelectTag(tagName) {
 			this.tags = this.tags.filter(tag => tag !== tagName);
 			localStorage.setItem(LOCAL_STORAGE_TAGS_KEY, JSON.stringify(this.tags));
+			this.updateUrl();
 		},
 		resetTags() {
 			this.tags = [];
 			localStorage.setItem(LOCAL_STORAGE_TAGS_KEY, '[]');
+			this.updateUrl();
+		},
+		updateUrl() {
```

Comment:
IMHO, 
I would prefer to have some button that is triggering url building by the click. And result url can be saved to clipboard ( we have such functionality for the "service sharing" feature ). I would like to avoid any real-time url changes.